src/ansible_navigator/image_manager/puller.py

```diff
@@ -38,15 +41,15 @@ def __init__(self, container_engine: str, image: str, pull_policy: str):
         self._image_tag: str
         self._logger = logging.getLogger(__name__)
         self._messages: List[LogMessage] = []
-        self._pull_policy = pull_policy
+        self._policy: str = policy
```

Comment:
Maybe do this change in a separate PR. It's a good rename, but it's unrelated to adding this argument.

Reply:
ok.